.github/workflows/ci_run_scm_rts.yml

```diff
@@ -172,5 +172,30 @@ jobs:
     - name: Run SCM RTs
       run: |
         cd ${SCM_ROOT}/scm/bin
-        mkdir ${RUN_DIR} && mkdir ${BIN_DIR}
-        ./run_scm.py --file /home/runner/work/ccpp-scm/ccpp-scm/test/rt_test_cases.py --runtime_mult 0.1
\ No newline at end of file
+        ./run_scm.py --file /home/runner/work/ccpp-scm/ccpp-scm/test/rt_test_cases.py --runtime_mult 0.1
+
+    - name: Gather SCM RT output
+      run: |
+        cd ${SCM_ROOT}/test
+        mkdir /home/runner/work/ccpp-scm/ccpp-scm/test/artifact-${{matrix.build-type}}
+        ./ci_util.py -b ${{matrix.build-type}}
+
+    - name: Create directory for SCM RT baselines
+      run: mkdir /home/runner/work/ccpp-scm/ccpp-scm/test/BL-${{matrix.build-type}}
+
+    - name: Download SCM RT baselines
+      run: |
+        cd /home/runner/work/ccpp-scm/ccpp-scm/test/BL-${{matrix.build-type}}
+        wget ftp://ftp.rap.ucar.edu:/incoming/irap/met_help/ccpp-scm/rt-baselines-${{matrix.build-type}}.zip
```

Comment:
Not sure that this is the best place on the server (`met_help`), but probably doesn't matter too much

Reply:
I've been waiting on RAL IT to create a directory for me on the FTP site...